src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -41,6 +45,72 @@ class WalletControllerTest {
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Test
+    void itShouldReturnStatusOKAndCorrectResponseBody() throws Exception {
+        //GIVEN
+        Instant timeCreated = Instant.now();
+
+        UpdateWalletDTO updateWalletDto = new UpdateWalletDTO(1L, "anyName");
+        given(walletService.updateWallet(Mockito.any())).willReturn(
+                new WalletDTO(
+                        updateWalletDto.id(),
+                        updateWalletDto.name(),
+                        timeCreated
+                )
+        );
+
+        //WHEN
+        ResultActions editResultActions = mockMvc.perform(
+                put("/api/wallet")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(Objects.requireNonNull(objectMapper.writeValueAsString(updateWalletDto)))
+        );
+
+        //THEN
+        editResultActions.andExpect(status().isOk())
+                .andExpect(jsonPath("$.id", Matchers.equalTo(updateWalletDto.id().intValue())))
+                .andExpect(jsonPath("$.name", Matchers.equalTo(updateWalletDto.name())));
+    }
+
+    @Test
+    void itShouldReturnBadRequestWhenNameIsEmpty() throws Exception {
+        //GIVEN
+        Instant timeCreated = Instant.now();
+
```

Comment:
Why there's an empty line?

Reply:
To have kind of 'sectioning' in the testing method. But line has been removed if that is a case.